README.md

```diff
@@ -256,18 +256,16 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // times out, requiring another attempt at authentication.
                 "state_timeout_in_seconds": 600,
 
-                // The name of the OIDC claim which provides the mapping of an
-                // OIDC user to an iRODS user account.
-                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
-                "irods_user_claim": "irods_username",
-
-                // The mapping of a user to the provided values. All values must
-                // be matched to map an OIDC user to an iRODS user account.
-                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
-                "user_attribute_mapping": {
-                    "irods_username": {
-                        "sub": "123-abc-456-xyz",
-                        "email": "rods_user@example.org"
+                // Defines relevant information related to the User Mapping plugin system.
+                // Allows for the selection and configuration of the plugin.
+                "user_mapping": {
+                    // The full path to the desired plugin to load.
+                    // See the section titled "Configure your 'user_mapping'" for more details on available plugins.
+                    "plugin_path": "/path/to/plugin/the_plugin.so",
+
+                    // The configuration information required by the selected plugin to execute properly.
+                    // See the section titled "Configure your 'user_mapping'" for more details on plugin configuration.
```

Comment:
This line will need to be updated if the section title changes.

Reply:
Updated.